### PR TITLE
Update news, add Hawaii PV Partners logo

### PR DESCRIPTION
**Pull request by cwhanse**

### CODE REVIEW

**Comment by cwhanse** [comment]
Sorry I forgot to branch before committing. In a hurry today.

File: index.md [hunk 1]
```diff
@@ -34,17 +35,10 @@ modeling tools which are primarily designed to predict future performance, rathe
...
--  Compare actual and modeled performance using performance ratios.
--  Account for snow and soiling in performance models.
-
-In the coming months, the project team will build out capabilities in the [pvlib-python](https://github.com/pvlib/pvlib-python.git)
-modeling library that support the use cases. The project team will also draft a business plan to ensure that the
-modeling services are available to the industry after the project period has ended.
+We are beta-testing the Solar Performance Insight tool, and are eager to hear
+your feedback and comments. See above for instructions to get your own account.
+Feedback can be emailed to info@solarperformanceinsight.org, or join the development
+community at https://github.com/solarperformanceinsight.
```
Comment (wholmgren) [inline index.md]:
```suggestion
community at [https://github.com/SolarPerformanceInsight/solarperformanceinsight](https://github.com/SolarPerformanceInsight/solarperformanceinsight).
```

File: index.md [hunk 2]
```diff
@@ -34,17 +35,10 @@ modeling tools which are primarily designed to predict future performance, rathe
...
--  Calculate performance from actual weather with user-selected performance models.
--  Compare actual and modeled performance using performance ratios.
--  Account for snow and soiling in performance models.
-
-In the coming months, the project team will build out capabilities in the [pvlib-python](https://github.com/pvlib/pvlib-python.git)
-modeling library that support the use cases. The project team will also draft a business plan to ensure that the
-modeling services are available to the industry after the project period has ended.
+We are beta-testing the Solar Performance Insight tool, and are eager to hear
+your feedback and comments. See above for instructions to get your own account.
+Feedback can be emailed to info@solarperformanceinsight.org, or join the development
```
Comment (wholmgren) [inline index.md]:
```suggestion
Feedback can be emailed to [info@solarperformanceinsight.org](mailto:info@solarperformanceinsight.org), or join the development
```

**Review by wholmgren — APPROVED** [review]
link suggestions below

**Pull request merged**